Samples/Use multiple voucher codes on one order/Resources/Litium.Accelerator.Mvc/Campaigns/MultipleVoucherCodeCondition.cs

```diff
@@ -0,0 +1,178 @@
+﻿using Litium.Application.Runtime;
+using Litium.Foundation.Modules.ECommerce.Carriers;
+using Litium.Foundation.Modules.ECommerce.Plugins.Campaigns.ConditionTypes;
+using Litium.Foundation.Modules.ECommerce.Plugins.Campaigns;
+using Litium.Foundation.Modules.ECommerce;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Web;
+using Litium.Foundation.Modules.ECommerce.Plugins.Campaigns.Conditions;
+
+namespace Litium.Accelerator.Mvc.Campaigns
+{
+	public class MultipleVoucherCodeCondition : CartConditionType, IOrderConfirmationHandler, IConditionRequireCampaignInfo
+	{
+		private VoucherCodeCondition.Data m_data;
+		private Action<object> m_dataPersister = null;
+		private const string PATH = "~/Litium/ECommerce/WebUserControls/Conditions/VoucherCodeConditionControl.ascx";
+
+		internal class VoucherCodeConverter : JsonConverter
```

Comment:
Is the converter still used?

Reply:
Not since we used the same Data class as the original condition. Nice catch!